css/global.scss

```diff
@@ -15,6 +15,7 @@
 }
 
 body {
+	margin-top: 220px;
```

Comment:
This will be applied to all pages. Have u checked all of them with this margin ?